### DISCUSSION TITLE
USB Kernel reuse code

### DISCUSSION THREAD

**Discussion by joe-hidden** [micronucleus/micronucleus#254] [Q&A] [answered]
i am struggling with this question

i would like to use the USB communication Functions in the Nucleus to save Memory in my Application. Can i call in the Nucleus to use the USB communication Functions, and if yes, how?

THX Joe

**Comment by cpldcpu** [comment, marked as answer]
Unfortunately that does not work, since the V-USB code in micronucleus has been altered.
